### PR TITLE
Fixed dropdown not translating in spanish issue for Featured Programs

### PR DESCRIPTION
**Pull request by kscali**
-Added a check for locale to display either Spanish name or English name of programs.
-Fixed formatting issues for esLint on withUserForm

Only relevant change is on line 938. Everything else is just formatting.

### CODE REVIEW

**Comment by leovcunha** [comment]
Hi Koy! I think you mean the dropdown is in client signup? I find no problems in the feature programs.

**Comment by kscali** [comment]
Yes, sorry. It is for the dropdown.

File: app/javascript/components/reusable/withUserForm.js [hunk 1]
```diff
@@ -878,8 +934,9 @@ const withUserForm = (WrappedComponent, schema, wrappedProps) => {
             errorText={ errors.programs }
             selectionRenderer={ this.selectionRenderer }
           >
-            { _.map(programs, ({ name, id }) => {
-              return <MenuItem key={ id } insetChildren checked={ _.indexOf(userPrograms, name) > -1 } value={ name } primaryText={ <span> { name } </span> } />;
+            { _.map(programs, ({ name, spanish_name, id }) => {
+              let programName = this.props.match.params[0] === ENGLISH ? name : spanish_name;
```
Comment (leovcunha) [inline app/javascript/components/reusable/withUserForm.js]: Koy, considering that we intend to translate tutoria soon to other languages, we should start avoiding referring the language names inside the code. Do you think of any way it's possible doing this already here ?

**Comment by kscali** [comment]
I think for now, since it's only two languages and one line of code, it's ok to just merge it so it can work in production since this is labelled high in our boards. There is going to be heavy refactoring for the code base once multi-language is implemented since we will also need to do database migrations since we have 4 models using english/spanish and will have to incorporate the other languages.

**Pull request merged**